tests_e2e/tests/lib/vm_extension.py

```diff
@@ -107,9 +107,7 @@ def enable(
                 extension_parameters
             ).result(timeout=_TIMEOUT))
 
-        if result.provisioning_state not in ('Succeeded', 'Updating'):
```

Comment:
I copied this check from the original prototype, but it has been producing intermittent failures. Depending on the timing, it can be Succeeded, Updating or just None. I double-checked that on extension failure the call to result() in the previous line will raise an exception anyway, so this check is actually not needed and now we just log an INFO.